tests/compile/fused_redeclare_T3111.pyx

```diff
@@ -0,0 +1,21 @@
+# ticket: 3111
+# mode: compile
+# tag: warnings
+
+ctypedef unsigned char npy_uint8
+ctypedef unsigned short npy_uint16
+
+
+ctypedef fused dtype_t:
+    npy_uint8
+
+ctypedef fused dtype_t_out:
+    npy_uint8
+    npy_uint16
+
+
+def foo(dtype_t[:] a, dtype_t_out[:, :] b):
+    pass
+
+
+_WARNINGS = ""
```

Comment:
The warnings are parsed line by line from the source file, so this would normally read
```python
_WARNINGS = """
"""
```